topology/probes/ovsdb/ovs_of.go

```diff
@@ -556,10 +826,11 @@ func (o *OvsOfProbe) OnOvsBridgeAdd(bridgeNode *graph.Node) {
 	bridgeName, _ := bridgeNode.GetFieldString("Name")
 	bridgeProbe, err := o.NewBridgeProbe(hostName, bridgeName, uuid, bridgeNode)
 	if err == nil {
-		logging.GetLogger().Infof("Probe added for %s on %s (%s)", bridgeName, hostName, uuid)
+		logging.GetLogger().Debugf("Probe added for %s on %s (%s)", bridgeName, hostName, uuid)
 		o.BridgeProbes[uuid] = bridgeProbe
 	} else {
 		logging.GetLogger().Errorf("Cannot add probe for bridge %s on %s (%s)", bridgeName, hostName, uuid)
```

Comment:
logging.GetLogger().Errorf("Cannot add probe for bridge %s on %s (%s) : %s", bridgeName, hostName, uuid, err)